core/services/wifi/main.py

```diff
@@ -9,6 +9,7 @@
 import uvicorn
 from commonwealth.utils.apis import PrettyJSONResponse
 from fastapi import FastAPI, HTTPException, status
+from fastapi.middleware.cors import CORSMiddleware
```

Comment:
this is not necessary, use nginx.

Reply:
As the [TEMP] states, that's there just for the tests, as nginx was not yet merged. Will remove after rebasing.